### ISSUE TITLE
Enable tethering doesn't work sometimes

### ISSUE THREAD

**Issue by adeebshihadeh** [commaai/openpilot#25141]
Seems to happen around 1/10 times. It's easy to tell because the IP doesn't change, and the tethering network doesn't show in the wifi list.

**Comment by adeebshihadeh** [comment]
@deanlee want to look into this one?

**Comment by bongbui321** [comment]
@adeebshihadeh . Hmm I have been trying to replicate the issue on my computer by running ./ui and enabling the tethering option only and didn't see the issue occur. Since this is in the `No hardware required ` section so I was experimenting on my computer only. This is how I was looking for the issue:

1.  Change the tethering_ssid in my to my hotspot name (instead of `weedle`)
2. Run `./ui` and enabling the tethering option
3. After toggling on/off about 20 times I still see the IP address consistently change to my Hotspot's IP address with no issue

does this issue only happen using the comma's data or with phone's data as well? Rn I can't recreate the issue by tethering using my phone's data

**Comment by adeebshihadeh** [comment]
I assumed this would be a general NetworkManager issue, but it might be a quirk of the comma three or its particular version of NetworkManager. I'll change the bounty to comma device.

**Comment by BBBmau** [comment]
Should this still be left open? I tried wednesday to reproduce it and just wasn't able to. I'm not even sure what the first step would be to reproduce this since repeatedly turning thethering on and off just isn't consistent.

**Comment by adeebshihadeh** [comment]
Yes, this is still an issue. I suspect it will only happen once per boot.

**Comment by royjr** [comment]
Can't seem to replicate this on demand, but it definitely does occur. Tried per boot, with or without a connected wifi network and or ignition.

**Comment by adeebshihadeh** [comment]
@sshane is this fixed?
